### PR TITLE
Add my author tags

### PR DESCRIPTION
**Pull request by KVignesh122**
Added author tags for portions I coded.

#118

### CODE REVIEW

File: src/main/java/seedu/mtracker/commons/error/InvalidNameError.java [hunk 1]
```diff
@@ -10,11 +10,13 @@ public InvalidNameError(String instrumentType) {
         this.instrumentType = instrumentType;
     }
 
+    //@@author KVignesh122
```
Comment (theodorekwok) [inline src/main/java/seedu/mtracker/commons/error/InvalidNameError.java:13]: I think u can just claim this whole file

File: src/main/java/seedu/mtracker/MTracker.java [hunk 2]
```diff
@@ -65,4 +65,5 @@ public void executeProgram() {
     public static void main(String[] args) {
         new MTracker().executeProgram();
     }
+    //@@author
```
Comment (theodorekwok) [inline src/main/java/seedu/mtracker/MTracker.java:68]: What is this ending author tag for? If I am not wrong it either checks the next author or end of file whichever comes first.  this is at end of file alr

File: src/main/java/seedu/mtracker/commons/error/InvalidNameError.java [hunk 3]
```diff
@@ -10,11 +10,13 @@ public InvalidNameError(String instrumentType) {
...
 
+    //@@author KVignesh122
     @Override
     public String getMessage() {
         if (instrumentType.equals(AddForexParser.INSTRUMENT_TYPE)) {
             return ErrorMessage.addForexNameError();
         }
         return ErrorMessage.addInstrumentNameError(instrumentType);
     }
+    //@@author
```
Comment (theodorekwok) [inline src/main/java/seedu/mtracker/commons/error/InvalidNameError.java:21]: Its at end of file, i dont think this is needed

File: src/main/java/seedu/mtracker/console/InputParser.java [hunk 4]
```diff
@@ -214,4 +217,5 @@ public static void checkIfAbort(String userInput, String currentProcess)
             throw new OperationAbortedError(currentProcess);
         }
     }
+    //@@author
```
Comment (theodorekwok) [inline src/main/java/seedu/mtracker/console/InputParser.java:220]: Same here

File: src/test/java/seedu/mtracker/console/GeneralInstrumentParserTest.java [hunk 5]
```diff
@@ -7,6 +7,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+//@@author KVignesh122
```
Comment (theodorekwok) [inline src/test/java/seedu/mtracker/console/GeneralInstrumentParserTest.java:10]: I would like to claim the simulateConsoleInput, checkParameters methods here if that is alright with you.

File: src/test/java/seedu/mtracker/console/AddStockParserTest.java [hunk 6]
```diff
@@ -76,6 +76,7 @@ void testStockParameters(String input, String[] expectedStockParameters) throws
         AddStockParser testStockParser = new AddStockParser();
         verifyInstrumentParameters(testStockParser, expectedStockParameters);
     }
+    //@@author
```
Comment (theodorekwok) [inline src/test/java/seedu/mtracker/console/AddStockParserTest.java:79]: I think the testXYZParameters method for the addXYZParserTest were written by us also.

**Review by theodorekwok — COMMENTED** [review]
Overall looks good, just not sure abt the ending //@@author tags near the end of the file. Cause if I am not wrong those would be redundant since there is no code below already.

File: src/test/java/seedu/mtracker/console/GeneralInstrumentParserTest.java (hunk 5 above)
Comment (KVignesh122) [reply src/test/java/seedu/mtracker/console/GeneralInstrumentParserTest.java:10]: Ok have included your github user name in author tag.

File: src/test/java/seedu/mtracker/console/AddStockParserTest.java (hunk 6 above)
Comment (KVignesh122) [reply src/test/java/seedu/mtracker/console/AddStockParserTest.java:79]: ok

File: src/main/java/seedu/mtracker/console/InputParser.java [hunk 7]
```diff
@@ -234,29 +251,6 @@ private void getAndValidateDoneStatus(String[] commandComponents, ArrayList<Inst
...
-    }
-
-    /**
-     * Filters and returns the command type based on the user input.
-     *
-     * @param commandComponents A string array containing the command words the user gave.
-     * @param instruments The list of instruments currently in the user's watchlist.
-     * @return The command the user wants to execute.
-     * @throws Exception If any of the user's addition or lack of inputs violates the command requirements.
-     */
```
Comment (theodorekwok) [inline src/main/java/seedu/mtracker/console/InputParser.java:259]: I dont think this was deleted by accident? Can help check

**Review by theodorekwok — COMMENTED** [review]
Looks good i think got one accidental delete

File: src/main/java/seedu/mtracker/console/InputParser.java (hunk 7 above)
Comment (KVignesh122) [reply src/main/java/seedu/mtracker/console/InputParser.java:259]: Oops so sorry. Don't know why it got deleted. Have added back in.

**Review by theodorekwok — APPROVED** [review]

**Pull request merged**